moped-database/views/project_geography.sql

```diff
@@ -1,25 +1,46 @@
--- latest version from 1681856452639_normalize_project_components_table
-DROP VIEW "public"."project_geography";
-
-CREATE OR REPLACE VIEW project_geography AS (
-    SELECT
-        moped_project.project_id AS project_id,
-        uniform_features.id AS feature_id,
-        moped_components.component_id AS component_archtype_id,
-        moped_proj_components.project_component_id AS component_id,
-        moped_proj_components.is_deleted,
-        moped_project.project_name AS project_name,
-        feature_layers.internal_table AS TABLE,
-        feature_layers.reference_layer_primary_key_column AS original_fk,
-        moped_components.component_name AS component_name,
-        uniform_features.attributes AS attributes,
-        uniform_features.geography AS geography
-    FROM
-        moped_project
-        JOIN moped_proj_components ON (moped_proj_components.project_id = moped_project.project_id)
-        JOIN moped_components ON (moped_proj_components.component_id = moped_components.component_id)
-        JOIN feature_layers ON (moped_components.feature_layer_id = feature_layers.id)
-        JOIN uniform_features ON (moped_proj_components.project_component_id = uniform_features.component_id)
-        --join component_feature_map on (moped_proj_components.project_component_id = component_feature_map.component_id)
-        --join uniform_features on (component_feature_map.feature_id = uniform_features.id)
-);
+-- latest version from 1691698449987_add-district-columns-to-views
+CREATE
+OR REPLACE VIEW "public"."project_geography" AS
+SELECT
+  moped_project.project_id,
+  uniform_features.id AS feature_id,
+  moped_components.component_id AS component_archtype_id,
+  moped_proj_components.project_component_id AS component_id,
+  moped_proj_components.is_deleted,
+  moped_project.project_name,
+  feature_layers.internal_table AS "table",
+  feature_layers.reference_layer_primary_key_column AS original_fk,
+  moped_components.component_name,
+  uniform_features.attributes,
+  uniform_features.geography,
+  uniform_features.council_districts
```

Comment:
this is the only change to this view

Reply:
looking at the diff below - do you use a SQL formatter? I've been looking for one of those in my life and could use a rec 🙏

Reply:
same! i still haven't found a formatter i like for VSCode, but [TablePlus](https://tableplus.com/) has a prettifier built into it's query interface, so i copypasta from that 😅